website/layouts/partials/_default/sidemenu.html

```diff
@@ -1,68 +1,149 @@
 {{ $currentPage := . }}
+{{- $currentVersion := index (split .File.Path "/") 1 }}
+{{- $versionDocsPath := printf "docs-pre-processed/%s" $currentVersion }}
+{{ $url := .RelPermalink }}
 
 <button class="btn btn-block btn-outline-secondary mb-3 d-block d-lg-none" type="button" data-toggle="collapse" data-target=".docs-menu">
     Toggle Menu
 </button>
 
-<nav class="docs-menu collapse d-lg-block">
+{{ range .Site.Sections }}
+
+{{ $docsPage := . }}
+{{ $urll := .RelPermalink}}
+ 
+<nav class="docs-menu collapse d-lg-block"> 
+
+  {{ range .Pages.GroupBy "Section" }}
+  {{ if in $url $urll }}
+  
+    
+      
+    {{ range .Pages.GroupByParam "menu" }}
+    {{ if eq .Key "thanos"}}
     <div class="sidemenu-thanos">
-        <input type="checkbox" id="menuinput-thanos" class="menuinput"{{if eq .Section ""}} checked{{end}} />
-        <label for="menuinput-thanos" class="menulabel-thanos">
-            <h5>Thanos</h5>
-            <img class="caret-expand" src="/expand.svg" alt="expand" />
-        </label>
-        <div class="list-group list-group-flush">
-            {{ range .Site.Menus.thanos }}
-                <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
-            {{ end }}
-        </div>
+      <input type="checkbox" id="menuinput-thanos" class="menuinput" {{if eq .Key ""}} checked {{end}}/>
```

Comment:
It would be amazing if we have no special treating like ` {{ if eq .Key "thanos"}}` for each section but just have `{{ range .Pages.GroupByParam "menu" }}` 

and then have generic side menu  content that will render properly for each menu.

Reply:
So we need to render:

`id="menuinput-thanos" ` => `id="menuinput-{{ <menu-name> }}" 

For this CSS class do we need special class for each menu item? (`list-group-item-thanos`) I don't think so :thinking:  same with `menulabel-thanos`

Reply:
Done!